src/main/resources/META-INF/resources/primefaces/ai/speaker/speaker.js

```diff
@@ -0,0 +1,77 @@
+PrimeFaces.widget.Speaker = PrimeFaces.widget.BaseWidget.extend({
+	
+	init: function(cfg) {
+    	this._super(cfg);
+        if(this.cfg.disabled) {
+            return;
+        }
+        
+        this.cfg.voice = this.cfg.voice || 'en-US';
+        this.cfg.pitch = this.cfg.pitch || 1;
+        this.cfg.rate = this.cfg.rate || 1;
+        
+	},
+	
+	speak: function(txt, voice){
+		
+        if (!txt) {
+        	return;
+        }
+        
+        var synth = window.speechSynthesis;
+        
+        if (!synth) {
+        	return;
+        }
+        if (synth.speaking) {
+        	return;
+        }
+        
+        var speech = new SpeechSynthesisUtterance(txt);
+        
+        speech.onend = function (event) {
+            console.log('SpeechSynthesisUtterance.onend');
```

Comment:
Shouldn't this be a `callback` hook so users can get the onerror?  Maybe even and AJAX event `callBehavior`